docs/BUILD.md

```diff
@@ -115,6 +115,14 @@ To build the code, you just need to run `build.ps1` in the `scripts` folder:
 ./scripts/build.ps1
 ```
 
+Note that in windows you will need to use Powershell 7 in administrator mode to create the builds and test.
```

Comment:
`build.ps1` shouldn't require running powershell as Administrator

Reply:
Tested without Administrator and it worked.